### DISCUSSION TITLE
SH1122 256x64 OLED on I2C

### DISCUSSION THREAD

**Discussion by spiderjuka** [olikraus/u8g2#1416] [Q&A]
Hello,

I need to buy some display 256x64 OLED which can work o I2C with your library.
Can you give me any link to such kind of display where I can buy it?
I already buy one on aliexpress but it doesn't work:
https://www.aliexpress.com/item/12832-2-08-inch-display-SSD1307-PM-OLED-256x64-SH1122-7pin-4-Wire-SPI-3-Wire/32865527538.html?spm=a2g0s.9042311.0.0.51495c0fKd2CyB
Maich choice was: No.10 Module 1 25664 White SH1122 7Pin SPI/IIC

Thank you very much for support.

**Comment by olikraus** [comment]
issue exists: https://github.com/olikraus/u8g2/issues/1414
